src/test/java/tutorease/address/testutil/CommandsUtil.java

```diff
@@ -0,0 +1,83 @@
+package tutorease.address.testutil;
+
+import static tutorease.address.logic.commands.CommandTestUtil.DURATION_DESC;
+import static tutorease.address.logic.commands.CommandTestUtil.FEE_DESC;
+import static tutorease.address.logic.commands.CommandTestUtil.START_DATE_TIME_DESC;
+import static tutorease.address.logic.commands.CommandTestUtil.STUDENT_ID_DESC;
+
+import tutorease.address.commons.core.index.Index;
+import tutorease.address.logic.commands.AddContactCommand;
+import tutorease.address.logic.commands.ContactCommand;
+import tutorease.address.logic.commands.DeleteContactCommand;
+import tutorease.address.logic.commands.EditContactCommand;
+import tutorease.address.logic.commands.FindContactCommand;
+import tutorease.address.logic.commands.LessonCommand;
+import tutorease.address.model.person.Person;
+
+/**
+ * A utility class for Commands.
+ */
+public class CommandsUtil {
+    /**
+     * Returns an uppercase add command string for adding the {@code person}.
+     *
+     * @param person Person object.
+     * @return String of the uppercase add command.
+     */
+    public static String getUpperCaseAddContactCommand(Person person) {
+        return ContactCommand.COMMAND_WORD.toUpperCase() + " " + AddContactCommand.COMMAND_WORD
+                + " " + PersonUtil.getPersonDetails(person);
+    }
+
+    /**
+     * Returns an uppercase delete command string for deleting {@code index}.
+     *
+     * @param index Index of the person to be deleted.
+     * @return String of the uppercase delete command.
+     */
+    public static String getUpperCaseDeleteContactCommand(Index index) {
+        return ContactCommand.COMMAND_WORD.toUpperCase() + " " + DeleteContactCommand.COMMAND_WORD
+                + " " + index.getOneBased();
+    }
+
+    /**
+     * Returns an uppercase edit command string for editing {@code index}.
+     *
+     * @param descriptor EditPersonDescriptor object.
```

Comment:
Should we include all parameters in the JavaDoc?

Reply:
Good catch!